bin/setup

```diff
@@ -1,29 +1,35 @@
 #!/usr/bin/env ruby
+
 require 'pathname'
+require 'fileutils'
+include FileUtils
```

Comment:
Is it necessary to include *and* require here

Reply:
removed include